libraries/AP_Motors/AP_MotorsHeli_Dual.cpp

```diff
@@ -260,11 +260,15 @@ void AP_MotorsHeli_Dual::set_desired_rotor_speed(float desired_speed)
 // calculate_armed_scalars
 void AP_MotorsHeli_Dual::calculate_armed_scalars()
 {
+    float thrcrv[5];
+    for (uint8_t i = 0; i < 5; i++) {
+    thrcrv[i]=_rsc_thrcrv[i]*0.001f;
```

Comment:
miss 4 spaces for indentation and should be thrcrv[i] = _rsc_thrcrv[i] * 0.001f;